tests/main.nf.test

```diff
@@ -29,22 +29,15 @@ nextflow_pipeline {
             assertAll(
                 { assert workflow.success },
                 { assert snapshot(
-                    path("${params.outdir}/VirusEnrichment"),
```

Comment:
Why are these removed? Have they been turned off in the defaul test?

Reply:
Yes they have been removed! I changed my default test to only run "essential" modules whereas it was previously a "full" test.